### ISSUE TITLE
Many already resolved Github issues are still open

### ISSUE THREAD

**Issue by camoz** [digint/btrbk#463]
I just went through some of the open github issues here, and it seems that many (like 50% ?) are already resolved and could be closed.

Is there a specific reason for keeping them open?

Random examples:

https://github.com/digint/btrbk/issues/113
https://github.com/digint/btrbk/issues/172
https://github.com/digint/btrbk/issues/218
https://github.com/digint/btrbk/issues/224
https://github.com/digint/btrbk/issues/403

**Comment by digint** [comment]
> Is there a specific reason for keeping them open?

No. Point is that people very rarely close the tickets themselves, and it's already a lot of work for me to answer all of the questions, double-checking which ones are resolved is simply too time-consuming, this war is kind of lost with hundreds of open tickets...

Also, I tend to leave the questions open, hoping that people find them before asking the same thing twice. Most of them are lazy and don't even search the issues, and the default search is "is:open". Not sure what the github best-practices are here, maybe you have a suggestion?

**Comment by camoz** [comment]
Ah, I see. Thanks for the explanation!

> Not sure what the github best-practices are here, maybe you have a suggestion?

I don't know if there are best practices...

What I see a lot is that github issues are used for actual "issues" only, like bugs and feature request, and not for support questions. That makes it easier for the developer(s) to have an overview over actual issues in the project that need to be addressed. Support issues are rigorously closed (and redirected to other communication channles)...

There is a relatively new feature (which is stable now) called [Github discussions](https://docs.github.com/en/discussions), which could be used for as a communication channle support questions. IMO this seems like a good option.

If there is interest in "cleaning up" the issues here and I find some free time, I'm happy to help out with this, even if it is only next year or so (I'm as well quite busy right now). I could compile a list of:

- all duplicates
- obsolete issues that have been addressed
- support questions, to move them somewhere else (if that policy is desired)
- etc.

... so that it's easy for you to double-check and handle them.
